admin/assets/javascripts/global-scripts.js

```diff
@@ -382,7 +382,7 @@ $(function () {
      * @returns {boolean}
      */
     isGroupNameComplete: function () {
-      var elName = $('input#name').val()
+      var elName = $('input#name').val().trim()
```

Comment:
I suspect that this won't work in IE8.

Reply:
🙄  I'll test it on IE8. However this is an edge case and it's not a critical fix because we still get a server side error on the screen.

Reply:
@jon-shipley - Tests passed and @sinhaamit1980 is happy with this PR. Can we please get this approved?

Reply:
Are we even supporting IE8. I can never remember 🤔

Reply:
@GuyHarwood - functional support for IE8-10

Reply:
@pris54 

Why is this a client-side fix and not server side?
We need to support IE functionally and calling methods that may not be implemented doesn't work for me.   You could use Jquery.trim() though?

Reply:
In that case safest, and likely easiest would be as Jon says, server side

Reply:
@pris54 

Fix it?

![screen shot 2018-02-14 at 14 00 50](https://user-images.githubusercontent.com/13945069/36208087-cc18ee10-118f-11e8-858c-87fe7ae3322e.png)